examples/src/main/java/com/mapbox/navigation/examples/core/BaseRouterActivityKt.kt

```diff
@@ -275,20 +273,9 @@ abstract class BaseRouterActivityKt :
         }
 
         fun setupOnboardRouter(accessToken: String, deviceProfile: DeviceProfile, context: Context): Router {
-            val file = File(
-                Environment.getExternalStoragePublicDirectory("Offline").absolutePath,
```

Comment:
need to add external storage option
https://github.com/mapbox/mapbox-navigation-android/pull/3020/files#r429473740